frontend/src/pages/lesson/InsightsDuring.jsx

```diff
@@ -0,0 +1,261 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import { observer } from 'mobx-react-lite';
+import Activity from './Activity';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Paper from '@material-ui/core/Paper';
+import PercentageDoughnut from '../_shared/PercentageDoughnut';
+import service from '../../service';
+
+import PaperWithHeader from '../_shared/PaperWithHeader';
+import ProgressBar from '../_shared/ProgressBar';
+
+const useStyles = makeStyles((theme) => ({
+  widget: {
+    padding: theme.spacing(3),
+  },
+  paper: {
+    borderBottomLeftRadius: 4,
+    borderBottomRightRadius: 4,
+    borderTopLeftRadius: 0,
+    borderTopRightRadius: 0,
+    height: 230,
+    padding: theme.spacing(3),
+  },
+}));
+
+const convertCategoryToString = (category) => {
+  const conversion = {
+    learning_process: 'Leerproces',
+    recognizing_morphemes_sentence: 'Herkennen morfemen in een zin',
+    meaning_morphemes: 'Betekenis morfemen',
+    splitsing_morphemes: 'Splits morfemen',
+    create_morphemes_prefix: 'Creëren morfemen (voorvoegsel)',
+    background_morphemes: 'Alternatieve betekenis morfemen',
+    recognizing_morphemes_text: 'Herkennen morfemen in een tekst',
+    intuition: 'Intuïtie',
+    create_alternative_morphemes: 'Creëren alternatieve morfemen',
+    create_morphemes_suffix: 'Creëren morfemen (achtervoegsel)',
+    create_new_morphemes: 'Creëren nieuwe morfemen',
+  };
+
+  return conversion[category];
+};
+
+const InsightsDuring = (props) => {
+  const { questionGroupIds, lessonId, classId, questionStore, studentStore } =
+    props;
+  const math = require('mathjs');
```

Comment:
dit graag bovenaan importe. Require gebruiken we eigenlijk alleen in de backend (heeft te maken met andere runtime/transpiler)